sdk/node/doc/classes/_hlc_.eventdeploysubmitted.html

```diff
@@ -0,0 +1,265 @@
+<!doctype html>
+<html class="default no-js">
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>EventDeploySubmitted | Hyperledger OpenBlockChain</title>
+	<meta name="description" content="">
```

Comment:
Many instances of `Hyperledger OpenBlockChain`. 
Change it to `Hyperledger Fabric`